ops-bedrock/docker-compose.yml

```diff
@@ -45,6 +45,7 @@ services:
       dockerfile: ./op-node/Dockerfile
     command: >
       op-node
+      start
```

Comment:
See comment about sub-command vs main command.

Reply:
Geth doesn't use required flags (the `Required` field in `cli.Command`), so subcommands can run without specifying the flag for the main command.